frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -348,3 +427,16 @@ function ScalarMetricsSection({ artifact }: ScalarMetricsSectionProps) {
     </div>
   );
 }
+async function getViewConfig(
+  v1VisualizationArtifact: LinkedArtifact | undefined,
+  namespace: string | undefined,
+): Promise<ViewerConfig[]> {
+  if (v1VisualizationArtifact) {
+    return OutputArtifactLoader.load(
+      WorkflowParser.parseStoragePath(v1VisualizationArtifact.artifact.getUri()),
+      namespace,
+    );
+  }
+  let v1ViewerConfigs: ViewerConfig[] = [];
+  return Promise.resolve(v1ViewerConfigs);
```

Comment:
nit: I recall that you can simplify to just

```
return [];
```

you might want to confirm whether that passes type checking

Reply:
Thank you for the reference! Makes sense, I will update the return value.